### PR TITLE
Adding test data

### LINKED ISSUE #40: Importer: More test data
We need more data for when we show the application off...

### PR DESCRIPTION
**Pull request by nchurchill**
this is the test data for the classes

Fixes #40

### CODE REVIEW

File: src/Athena.Importer/data/offering/ecnm152-p100-f2018.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "id": "c22cd6a4-b7f2-437e-bf45-09e774ae97cd",
```
Comment (nlowe) [inline src/Athena.Importer/data/offering/ecnm152-p100-f2018.json]: An offering with the same ID exists for `@{id=8588ca76-eaa0-43d0-bead-ad1c3374a881; name=MTHM  280 - LINEAR ALGEBRA;}`

File: src/Athena.Importer/data/offering/mthm280-p100-f2018.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+  "id": "c22cd6a4-b7f2-437e-bf45-09e774ae97cd",
```
Comment (nlowe) [inline src/Athena.Importer/data/offering/mthm280-p100-f2018.json]: An offering with the same ID exists for `@{id=9dc68c84-69fa-4ace-b8da-d08b44f0cc82; name= ECNM 152 - MICROECONOMICS;}`

File: src/Athena.Importer/data/requirement/college-physics-i.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "id": "890c78c1-5b8c-448c-8b08-57a78ee37504",
```
Comment (nlowe) [inline src/Athena.Importer/data/requirement/college-physics-i.json]: Multiple requirements with this ID exist:

```powershell
id                                   name
--                                   ----
890c78c1-5b8c-448c-8b08-57a78ee37504 COLLEGE PHYSICS I
890c78c1-5b8c-448c-8b08-57a78ee37504 COLLEGE PHYSICS II
890c78c1-5b8c-448c-8b08-57a78ee37504 EECS Data Communications
890c78c1-5b8c-448c-8b08-57a78ee37504 EECS Electric Circuits I
890c78c1-5b8c-448c-8b08-57a78ee37504 EECS First Year Design 
```

File: src/Athena.Importer/data/requirement/college-physics-ii.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+  "id": "890c78c1-5b8c-448c-8b08-57a78ee37504",
```
Comment (nlowe) [inline src/Athena.Importer/data/requirement/college-physics-ii.json]: Multiple requirements with this ID exist:

```powershell
id                                   name
--                                   ----
890c78c1-5b8c-448c-8b08-57a78ee37504 COLLEGE PHYSICS I
890c78c1-5b8c-448c-8b08-57a78ee37504 COLLEGE PHYSICS II
890c78c1-5b8c-448c-8b08-57a78ee37504 EECS Data Communications
890c78c1-5b8c-448c-8b08-57a78ee37504 EECS Electric Circuits I
890c78c1-5b8c-448c-8b08-57a78ee37504 EECS First Year Design 
```

File: src/Athena.Importer/data/requirement/eecs-data-communications.json [hunk 5]
```diff
@@ -0,0 +1,5 @@
+{
+  "id": "890c78c1-5b8c-448c-8b08-57a78ee37504",
```
Comment (nlowe) [inline src/Athena.Importer/data/requirement/eecs-data-communications.json]: Multiple requirements with this ID exist:

```powershell
id                                   name
--                                   ----
890c78c1-5b8c-448c-8b08-57a78ee37504 COLLEGE PHYSICS I
890c78c1-5b8c-448c-8b08-57a78ee37504 COLLEGE PHYSICS II
890c78c1-5b8c-448c-8b08-57a78ee37504 EECS Data Communications
890c78c1-5b8c-448c-8b08-57a78ee37504 EECS Electric Circuits I
890c78c1-5b8c-448c-8b08-57a78ee37504 EECS First Year Design 
```

File: src/Athena.Importer/data/requirement/eecs-electric-circuits-i.json [hunk 6]
```diff
@@ -0,0 +1,5 @@
+{
+  "id": "890c78c1-5b8c-448c-8b08-57a78ee37504",
```
Comment (nlowe) [inline src/Athena.Importer/data/requirement/eecs-electric-circuits-i.json]: Multiple requirements with this ID exist:

```powershell
id                                   name
--                                   ----
890c78c1-5b8c-448c-8b08-57a78ee37504 COLLEGE PHYSICS I
890c78c1-5b8c-448c-8b08-57a78ee37504 COLLEGE PHYSICS II
890c78c1-5b8c-448c-8b08-57a78ee37504 EECS Data Communications
890c78c1-5b8c-448c-8b08-57a78ee37504 EECS Electric Circuits I
890c78c1-5b8c-448c-8b08-57a78ee37504 EECS First Year Design 
```

File: src/Athena.Importer/data/requirement/eecs-first-year-design.json [hunk 7]
```diff
@@ -0,0 +1,5 @@
+{
+  "id": "890c78c1-5b8c-448c-8b08-57a78ee37504",
```
Comment (nlowe) [inline src/Athena.Importer/data/requirement/eecs-first-year-design.json]: Multiple requirements with this ID exist:

```powershell
id                                   name
--                                   ----
890c78c1-5b8c-448c-8b08-57a78ee37504 COLLEGE PHYSICS I
890c78c1-5b8c-448c-8b08-57a78ee37504 COLLEGE PHYSICS II
890c78c1-5b8c-448c-8b08-57a78ee37504 EECS Data Communications
890c78c1-5b8c-448c-8b08-57a78ee37504 EECS Electric Circuits I
890c78c1-5b8c-448c-8b08-57a78ee37504 EECS First Year Design 
```

File: src/Athena/Views/Home/Add_Classes.cshtml [hunk 8]
```diff
@@ -22,7 +22,7 @@
                 <li>
```
Comment (nlowe) [inline src/Athena/Views/Home/Add_Classes.cshtml]: What are these changes for?

File: src/Athena.Importer/data/course/ecnm151.json [hunk 9]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "91db8c65-c6e5-4a9d-8738-0d865f22c364",
+  "name": " ECNM 151 - MACROECONOMICS",
```
Comment (nlowe) [inline src/Athena.Importer/data/course/ecnm151.json]: Strip extra space from name

File: src/Athena.Importer/data/course/ecnm152.json [hunk 10]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "9dc68c84-69fa-4ace-b8da-d08b44f0cc82",
+  "name": " ECNM 152 - MICROECONOMICS",
```
Comment (nlowe) [inline src/Athena.Importer/data/course/ecnm152.json]: Strip extra space from name

File: src/Athena.Importer/data/course/mthm270.json [hunk 11]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "ca874843-d2dc-404d-aab1-f284032417e9",
+  "name": "MTHM  270 - DISCRETE MATHEMATICS",
```
Comment (nlowe) [inline src/Athena.Importer/data/course/mthm270.json]: Strip extra space from name (Between `MTHM` and `270`)

File: src/Athena.Importer/data/course/mthm280.json [hunk 12]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "8588ca76-eaa0-43d0-bead-ad1c3374a881",
+  "name": "MTHM  280 - LINEAR ALGEBRA",
```
Comment (nlowe) [inline src/Athena.Importer/data/course/mthm280.json]: Strip extra space from name (Between `MTHM` and `280`)

**Comment by nlowe** [comment]
@nchurchill I think we can vastly simplify this by removing the `meetings` category, since they only exist in the context of an `offering`. The importer would need changed a bit but I think we could make it work. Thoughts?

File: src/Athena.Importer/data/course/ecnm151.json (hunk 9 above)
Comment (nchurchill) [reply src/Athena.Importer/data/course/ecnm151.json]: done

File: src/Athena.Importer/data/offering/ecnm152-p100-f2018.json (hunk 1 above)
Comment (nchurchill) [reply src/Athena.Importer/data/offering/ecnm152-p100-f2018.json]: ok

File: src/Athena.Importer/data/offering/mthm280-p100-f2018.json (hunk 2 above)
Comment (nchurchill) [reply src/Athena.Importer/data/offering/mthm280-p100-f2018.json]: ok

File: src/Athena.Importer/data/requirement/college-physics-i.json (hunk 3 above)
Comment (nchurchill) [reply src/Athena.Importer/data/requirement/college-physics-i.json]: done

File: src/Athena.Importer/data/requirement/college-physics-ii.json (hunk 4 above)
Comment (nchurchill) [reply src/Athena.Importer/data/requirement/college-physics-ii.json]: done

File: src/Athena.Importer/data/requirement/eecs-data-communications.json (hunk 5 above)
Comment (nchurchill) [reply src/Athena.Importer/data/requirement/eecs-data-communications.json]: done

File: src/Athena.Importer/data/requirement/eecs-electric-circuits-i.json (hunk 6 above)
Comment (nchurchill) [reply src/Athena.Importer/data/requirement/eecs-electric-circuits-i.json]: done

File: src/Athena.Importer/data/requirement/eecs-first-year-design.json (hunk 7 above)
Comment (nchurchill) [reply src/Athena.Importer/data/requirement/eecs-first-year-design.json]: done

File: src/Athena/Views/Home/Add_Classes.cshtml (hunk 8 above)
Comment (nchurchill) [reply src/Athena/Views/Home/Add_Classes.cshtml]: i was trying to show peter the thing

File: src/Athena.Importer/data/course/ecnm152.json (hunk 10 above)
Comment (nchurchill) [reply src/Athena.Importer/data/course/ecnm152.json]: DONE

File: src/Athena.Importer/data/course/mthm270.json (hunk 11 above)
Comment (nchurchill) [reply src/Athena.Importer/data/course/mthm270.json]: DONE

File: src/Athena.Importer/data/course/mthm280.json (hunk 12 above)
Comment (nchurchill) [reply src/Athena.Importer/data/course/mthm280.json]: DONE

File: src/Athena.Importer/data/course/mthm270.json (hunk 11 above)
Comment (nlowe) [reply src/Athena.Importer/data/course/mthm270.json]: Lies, it's still in the diff :)

File: src/Athena.Importer/Athena.Importer.csproj [hunk 13]
```diff
@@ -17,4 +17,5 @@
   <ItemGroup>
     <None Include="data/**/*.json" />
   </ItemGroup>
+  <ProjectExtensions><VisualStudio><UserProperties data_4meeting_4eecs3550-805-s2018-r_1json__JSONSchema="http://json.schemastore.org/bower" /></VisualStudio></ProjectExtensions>
```
Comment (nlowe) [inline src/Athena.Importer/Athena.Importer.csproj]: Where did this come from?

File: src/Athena/Views/Home/Add_Classes.cshtml (hunk 8 above)
Comment (nlowe) [reply src/Athena/Views/Home/Add_Classes.cshtml]: Let's revert this file then.

File: src/Athena.Importer/data/requirement/differential-equations.json [hunk 14]
```diff
@@ -0,0 +1,5 @@
+{
+  "id": "c67960f3-02ff-40ea-a77c-2e4c990102f8",
+  "name": "Differential Equations",
+  "description": "A class about different equations"
```
Comment (PeterETschaekofske) [inline src/Athena.Importer/data/requirement/differential-equations.json]: Wow. How descriptive.

**Pull request merged**